setup/kubernetes/aws.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Set Up an Amazon Elastic Kubernetes Cluster
+description: Learn how to set up an Amazon EKS cluster for your Coder deployment.
+---
+
+This deployment guide shows you how to set up an Amazon Elastic Kubernetes
+Engine cluster on which Coder can deploy.
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the [eksctl command line
+utility](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
+installed on your machine.
```

Comment:
```suggestion
Before proceeding, please make sure that you have the [eksctl command line
utility](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
and the [aws command line utility](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
installed on your machine.
```

This is a good port, but we will also need to have the aws cli, I put a suggested change, but please feel free to modify it as needed.

Reply:
My suggestion may be off on the 80 char line limit, webUI makes that a little hard sometimes.